build_library/vm_image_util.sh

```diff
@@ -610,6 +621,16 @@ _run_box_fs_hook() {
     sudo rm -fr "${VM_TMP_ROOT}/oem/box"
 }
 
+_run_ionoscloud_fs_hook() {
+    # Prepare root partition for IONOS Cloud legacy injection
+    # This is a workaround until the IONOS Cloud introduces a metadata server
```

Comment:
@tuunit can you put a date on when a metadata server will be introduced by IONOS Cloud? I wouldn't want this workaround to stay around forever, as it is likely to interfere with evolutions to the rootfs.

Reply:
Unfortunately not, I can ask again internally but it will most definitely stay like this for a long time. Up to a year or longer.